.scratch/authenticated-workspace-ui-stack/spec.md

```
@@ -0,0 +1,92 @@
+Status: ready-for-agent
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)